README.md

```diff
@@ -54,6 +54,23 @@ If you want to get fancy, both send job methods take `metadata`, `callback_url`,
 `skip_diarization`, `skip_punctuation`, `speaker_channels_count`, `custom_vocabularies`, `filter_profanity`, `remove_disfluencies`, `delete_after_seconds`, `language`, and `custom_vocabulary_id` as optional parameters, these are described in the request body of
 the [Submit Job](https://www.rev.ai/docs#operation/SubmitTranscriptionJob) endpoint.
 
+If you want transcription to be performed by a human, both methods allow you to submit human transcription jobs
+using `transcriber=human` with `verbatim`, `rush`, `segments_to_transcribe` and `test_mode` as optional parameters.
+Check out our documentation for [Human Transcription](https://www.rev.ai/docs#section/Human-Transcription-(Labs)) for more details.
+
+```python
+# submitting a human transcription jobs
+job = client.submit_job_url("https://example.com/file-to-transcribe.mp3",
+    transcriber='human',
+    verbatim=False,
+    rush=False,
+    test_mode=True
+    segments_to_transcribe=[{
```

Comment:
What do you think of having a segment object instead of this being a list of dictionaries

Reply:
I **_think_** the pattern in python is not to use objects but to just rely on type hints in `apiclient.py`

Reply:
@gweng88 can you give an example or link to type hints?

Reply:
I did a bit of research and the decision to use objects and the decision to use type hints seem pretty separate. Here's some of the things I looked at: 
- [Google's python style guide note on type hints](https://google.github.io/styleguide/pyguide.html#221-type-annotated-code)
- [AWS boto3 uses dicts for most user facing functions](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client.put_item) when you could argue they could use objects.
- [SDKs generated from openapi specs](https://github.com/plaid/plaid-python/blob/master/plaid/model/item_public_token_exchange_request.py#L28) expose objects to users (this is probably the only way the generation can work)

Reply:
Seems reasonable, lets leave it as is